### DISCUSSION TITLE
one to one relation in between wind speed and wind direction

### DISCUSSION THREAD

**Discussion by semira123** [NatLabRockies/floris#732] [Q&A]
Dears,

In case of data from the Metmast available, which includes the wind speed,  wind direction, and turbulent intensity is also known. Assume the yaw angle is zero, the data was recorded for 1 month, every 10min (4320 data). How to calculate the power that could be generated from the wind farm, every 10min, for 1 month, if the wind farm has two turbines.

Best,

**Comment by rafmudaf** [comment]
Hi @semira123 thanks for reaching out. I suggest you look through the examples to get started. A full listing of the examples at https://nrel.github.io/floris/examples.html, and the example scripts themselves are available in the repository on GitHub https://github.com/NREL/floris/tree/main/examples. I recommend to read through the first 6 to understand FLORIS, and then move to 21 for your case. 

Try to create a simple script to get started and then post back here with your progress.

**Comment by semira123** [comment]
Thanks a lot.
